governance/approveDeny/scripts/rundocker.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -it --name rt1 -p 8081:8081 -p 8082:8082 -p 5555:5555 -e "JAVA_TOOL_OPTIONS=\"-agentlib:jdwp=transport=dt_socket,address=*:5555,server=y,suspend=n\"" docker.bintray.io/jfrog/artifactory-pro:latest /bin/bash
+docker run -it --rm --name rt1 -p 8081:8081 -p 8082:8082 -p 5555:5555 -e "JAVA_TOOL_OPTIONS=\"-agentlib:jdwp=transport=dt_socket,address=*:5555,server=y,suspend=n\"" -v $(pwd)/approveDeny.groovy:/opt/jfrog/artifactory/var/etc/artifactory/plugins/approveDeny.groovy -v $(pwd)/approveDeny.json:/opt/jfrog/artifactory/var/etc/artifactory/plugins/approveDeny.json docker.bintray.io/jfrog/artifactory-pro:latest /bin/bash
```

Comment:
I thought bintray was going away

Reply:
Also, I think you should use docker-compose for this - it's meant to capture extensive docker config

Reply:
Good catch, I'll try to find the equivalent and update the URL.